docs/flashbots-mev-share/for-users.mdx

```diff
@@ -7,12 +7,4 @@ import WhatIsMevShare from "@site/docs/specs/mev-share/blurbs/_whatsMevShareBasi
 
 <WhatIsMevShare />
 
-Connecting your wallet to Flashbots Protect will automatically route your transactions to MEV-Share and enable you to capture your transactions' MEV.
-
-Please see the [Flashbots Protect Documentation](/flashbots-protect/mev-share) for more information.
-
-<ProtectButtonSelector />
-
-<br />
-
-
+MEV-Share Nodes generally accept transactons using the `eth_sendRawTransaction` or `eth_sendPrivateTransaction` json rpc requests. Flashbots runs a MEV-Share Node that can be used by sending transactions to [Flashbots Protect](/flashbots-protect/rpc/quick-start).
```

Comment:
If this is intended to be user facing, writing out the APIs seems a bit overly technical. I might lead with protect info.